contracts/extensions/OneTxPayment.sol

```diff
@@ -18,24 +18,40 @@
 pragma solidity 0.5.8;
 pragma experimental ABIEncoderV2;
 
-import "./../ColonyAuthority.sol";
-import "./../ColonyDataTypes.sol";
-import "./../IColony.sol";
-import "./../IColonyNetwork.sol";
+import "../ColonyAuthority.sol";
+import "../ColonyDataTypes.sol";
+import "../IColonyNetwork.sol";
+import "./ColonyExtension.sol";
 
 
-contract OneTxPayment {
+contract OneTxPayment is ColonyExtension {
   bytes4 constant ADD_PAYMENT_SIG = bytes4(keccak256("addPayment(uint256,uint256,address,address,uint256,uint256,uint256)"));
   bytes4 constant MOVE_FUNDS_SIG = bytes4(keccak256("moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,address)"));
 
-  IColony colony;
   IColonyNetwork colonyNetwork;
 
-  constructor(address _colony) public {
+  /// @notice Returns the version of the extension
+  function version() public pure returns (uint256) {
+    return 1;
+  }
+
+  /// @notice Configures the extension
+  /// @param _colony The colony in which the extension holds permissions
+  function install(address _colony) public auth {
+    require(address(colony) == address(0x0), "extension-already-installed");
+
     colony = IColony(_colony);
     colonyNetwork = IColonyNetwork(colony.getColonyNetwork());
   }
 
+  /// @notice Called when upgrading the extension (currently a no-op since this OneTxPayment does not support upgrading)
```

Comment:
I mean, this not being a noop doesn't mean upgrades aren't possible. I wouldn't expect any first version of an extension to have anything here, because for a version 1 there's no way this can be called.